core/templates/pages/exploration-player-page/services/extract-image-filenames-from-model.service.ts

```diff
@@ -191,26 +203,39 @@ export class ExtractImageFilenamesFromModelService {
     let collapsibleTagList = dummyDocument.getElementsByTagName(
       'oppia-noninteractive-collapsible');
     for (let i = 0; i < collapsibleTagList.length; i++) {
-      let contentWithValue = JSON.parse(
-        collapsibleTagList[i].getAttribute('content-with-value'));
-      let collapsibleDocument = (
-        new DOMParser().parseFromString(contentWithValue, 'text/html'));
-      imageTagLists.push(
-        collapsibleDocument.getElementsByTagName('oppia-noninteractive-image'));
+      // Created an attribute variable that gets the given attribute from
+      // a collapsible. If the given attribute does not exist, then attribute
+      // is null which means skip the current collapsible. If attribute is not
+      // null, then add images thatare embedded in the given collapsible.
```

Comment:
```suggestion
      // Get attribute from a collapsible. If the given attribute does not exist,
      // then attribute is null which means skip the current collapsible.
      // If the attribute is not null, then add images that are embedded in 
      // the given collapsible.
```

Change the other comments similarly.

Reply:
Done